docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md

```diff
@@ -74,6 +74,7 @@ This endpoint URL can then be used to create and manage custom objects.
 The `kind` of these objects will be `CronTab` from the spec of the
 CustomResourceDefinition object you created above.
 
+Please note that it might take sometime for the endpoint to be created.
```

Comment:
maybe add some detail like: Please note that it might take a secon dor two for the endpoint to be created. You can watch the `Established` condition on your CustomResourceDefinition to be true or watch the discovery information (link) of the API server for your resource to show up.

Reply:
Updated. Having a little trouble coming up with a good docs link for discovery information.